typings/react-native-code-push.d.ts

```diff
@@ -260,7 +260,7 @@ declare namespace CodePush {
      *
      * @param updateState The state of the update you want to retrieve the metadata for. Defaults to UpdateState.RUNNING.
      */
-    function getUpdateMetadata(updateState?: UpdateState) : Promise<LocalPackage>;
+    function getUpdateMetadata(updateState?: UpdateState) : Promise<null|LocalPackage>;
```

Comment:
@iamolegga,
Just small nit: Could you please put `LocalPackage` before `null` as `LocalPackage` is main type.

Reply:
@iamolegga, just a friendly ping :)